deployment/v3/utils/prop_migrator/Overview.md

```diff
@@ -137,13 +137,71 @@ This section lists the property files that were updated during the process. It i
 
 The script generates a log file (`logs.txt`) that provides detailed information about the update process. It includes information about matching combinations, unresolved combinations, similar latest values, and more. The log file helps you understand the changes made to the property files and allows you to review the update process.
 
-### Manual Configuration CSV
+### Manual Configuration CSV File - "manual-configuration.csv"
 
 The script creates or updates the `manual-configuration.csv` file. This file includes unresolved combinations and combinations that require further manual configuration. You can use this file to manually update the property files based on the required configuration actions mentioned in the file.
 
+The "manual-configuration.csv" file is a CSV (Comma-Separated Values) file that serves as a configuration guide for updating property files in the latest config branch. It contains information about specific combinations of properties found in different versions of config, where automatic resolution is not possible, and manual intervention is required.
+
+#### File Format :-
+The file has the following columns:
+
+* Property file name: The name of the property file where the property is defined.
+* Key: The key (or property name) that identifies the property.
+* Old Value: The value of the property in the old version of config.
+* Latest Value: The value of the property in the latest version of config.
+* Configuration Action: A comment indicating the required manual action for the property.
+  
+#### Comments Used in Configuration Action:
+The "Configuration Action" column includes comments that guide developers on how to handle specific combinations of properties during updates. The following comments are used in the file:
+
+#### 1. update-if-old-value-takes-priority:
+
+Description:
+This comment suggests that in the specific combination of properties, the value from the old version of config should take priority over the value in the latest version. In other words, the old value is considered more important or appropriate for the current context.
```

Comment:
this has come out well, we have to include one more detail on what situation this action will be set. Like the following.

This comment suggest that we have the same property in both old and the latest property files with different values, as a default behaviour utility has retained the latest property file value. If the old property file value is considered more appropriate, please manually update the same.

Reply:
Follow a similar explation in the description for the all the below actions also

Reply:
updated sir.